docs/product/components/textarea.html

```diff
@@ -8,13 +8,13 @@
     {% header "h2", "Base style" %}
     <div class="stacks-preview">
 {% highlight html %}
-<div class="d-flex fd-column gs4 gsy">
+<div class="d-flex fd-column gy4">
```

Comment:
### Changes requested

We should determine how we want to handle `p.s-input-message` both in the example code blocks as well as the rendered example. I think adding `.mb0` to the paragraph element will suffice.

Reply:
👍